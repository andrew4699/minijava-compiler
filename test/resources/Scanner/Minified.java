public class Mini {public static void main(String[] args){boolean t=false;boolean f=true;return t&&f;}}