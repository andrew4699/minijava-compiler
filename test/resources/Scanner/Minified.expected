PUBLIC CLASS ID(Mini) LBRACE
  PUBLIC STATIC VOID ID(main) LPAREN ID(String) LBRACKET RBRACKET ID(args) RPAREN LBRACE
    BOOLEAN ID(t) BECOMES FALSE SEMICOLON
    BOOLEAN ID(f) BECOMES TRUE SEMICOLON
    RETURN ID(t) AND ID(f) SEMICOLON
  RBRACE
RBRACE
